db/migrate/data/20160127210624_convert_configurations_to_settings_changes/vmdb.tmpl.yml

```diff
@@ -332,6 +332,8 @@ workers:
         :amqp_heartbeat: 30
       :event_catcher_amazon:
         :poll: 15.seconds
+      :event_catcher_hawkular:
+        :poll: 15.seconds
```

Comment:
@durandom yes, it does, a log snippet of each invocation...
```
[----] I, [2016-04-08T09:37:46.248846 #13139:8689aa4]  INFO -- : Catching Events since [1460120456000]
[----] I, [2016-04-08T09:38:01.378923 #13139:8603c74]  INFO -- : Catching Events since [1460120456000]
[----] I, [2016-04-08T09:38:16.550722 #13139:85db814]  INFO -- : Catching Events since [1460120456000]
[----] I, [2016-04-08T09:38:31.738954 #13139:85b18c0]  INFO -- : Catching Events since [1460120456000]
```

Although the mechanism works as I wrote it I will adopt the approach you pointed out because it seems to be the convention in many of the existing event catchers.